spring-integration-jdbc/src/test/java/org/springframework/integration/jdbc/store/channel/TestMailMessage.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2025-present the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.jdbc.store.channel;
+
+/**
+ * Test payload class for Jackson JSON serialization tests.
+ *
+ * @author Yoobin Yoon
+ */
+public record TestMailMessage(
```

Comment:
`@since 7.0` and not `public`.
I don't want test data classes to leak from one package to another.